thoth/storages/graph/postgres.py

```diff
@@ -342,7 +326,7 @@ def normalize_python_package_version(package_version: str) -> str:
     def parse_python_solver_name(solver_name: str) -> dict:
         """Parse os and Python identifiers encoded into solver name."""
         if solver_name.startswith("solver-"):
-            solver_identifiers = solver_name[len("solver-"):]
+            solver_identifiers = solver_name[len("solver-") :]
```

Comment:
black failing :/ also below

Reply:
thanks for pointing it out, fixing these black issues in another PR. :)